packages/web/src/components/notification/Notification/ChallengeRewardNotification.tsx

```diff
@@ -48,10 +43,7 @@ export const ChallengeRewardNotification = (
   const record = useRecord()
 
   const { title, icon } = getChallengeConfig(challengeId)
-  const { amount } = useSelector((state) =>
-    getUserChallenge(state, { challengeId })
-  )
-
+  const amount = stringWeiToAudioBN(notification.amount)
```

Comment:
Fwiw there's a new helper for converting different currencies to ui strings and vice versa, I think it would be something like `AUDIO(notification.amount).value` here. But not super necessary I think this is good!